src/blockchain_db/lmdb/db_lmdb.cpp

```diff
@@ -5311,7 +5307,6 @@ void BlockchainLMDB::migrate_2_3()
         if (result)
           throw0(DB_ERROR(lmdb_error("Failed to open a cursor for block_info: ", result).c_str()));
         if (!i) {
-          MDB_stat db_stat;
```

Comment:
That one was not unused. Removing it will instead cause a write to another in a previous scope. Which happens to be fine in practice in this case though.

Reply:
I believe this one was actually unused, it appears to be using `db_stats` everywhere, never `db_stat`

Reply:
Indeed, I did ready too quickly :)